cylc/uiserver/data_store_mgr.py

```diff
@@ -141,11 +143,11 @@ async def register_workflow(self, w_id):
         self.data[w_id] = data
 
         # create new entry in the delta store
-        self.update_contact(w_id)
+        self.update_contact(w_id, status='installed')
```

Comment:
Should we update the [`WorkflowStatus`](https://github.com/cylc/cylc-flow/blob/3537c161be43ddae7a1aa5373de14445ce159074/cylc/flow/workflow_status.py#L40) to include `installed` there too?

It's weird that we have `status=WorkflowStatus.STOPPED.value` (below) but not `status=WorkflowStatus.INSTALLED.value` here.

Reply:
Good point, I'll actually subclass it at the UIS (unless there's some reason to have it at the Scheduler.

Reply:
Moved to `cylc.flow` .. Sibling PR https://github.com/cylc/cylc-flow/pull/4206